sections/tooling/syntax-highlighting.js

```diff
@@ -0,0 +1,45 @@
+import md from 'components/md'
+
+const SyntaxHighlighting = () => md`
+  ## Syntax Highlighting
+
+  The one thing you lose when writing CSS in template literals is syntax highlighting. We're working hard on making proper syntax highlighting happening in all editors. We currently have support for Atom, Visual Studio Code, WebStorm, and soon Sublime Text.
```

Comment:
After reading the first sentence it feels like we are losing it now. Maybe slightly change it.

Reply:
Maybe put a past tense on it, good idea :)